apps/create-fuels-counter-guide/src/app/script/page.tsx

```diff
@@ -31,7 +32,9 @@ export default function ScriptExample() {
         return toast.error("Script not loaded");
       }
 
-      const { value } = await script.functions.main(bn(input)).call();
+      // Call the script with the input value
+      const { waitForResult } = await script.functions.main(bn(input)).call();
+      const { value } = await waitForResult();
```

Comment:
Looks like we forgot to make an adjustment to this piece of code when the API for `call()` was affected. I have fixed it here since I saw it